OctoKit/User.swift

```diff
@@ -2,22 +2,21 @@ import Foundation
 import RequestKit
 
 // MARK: model
-
-@objc open class User: NSObject, Codable {
-    @objc open internal(set) var id: Int = -1
-    @objc open var login: String?
-    @objc open var avatarURL: String?
-    @objc open var gravatarID: String?
-    @objc open var type: String?
-    @objc open var name: String?
-    @objc open var company: String?
-    @objc open var blog: String?
-    @objc open var location: String?
-    @objc open var email: String?
+open class User: Codable {
+    open internal(set) var id: Int = -1
+    open var login: String?
+    open var avatarURL: String?
+    open var gravatarID: String?
+    open var type: String?
+    open var name: String?
+    open var company: String?
+    open var blog: String?
+    open var location: String?
+    open var email: String?
     open var numberOfPublicRepos: Int?
     open var numberOfPublicGists: Int?
     open var numberOfPrivateRepos: Int?
-
+    
```

Comment:
Could you remove the whitespace?

Reply:
Sure

Reply:
fixed in https://github.com/nerdishbynature/octokit.swift/pull/69/commits/4ac9946121c9e559ecebdc1058addc10aeb484c7